mantle/rbx_mantle/src/roblox_resource_manager.rs

```diff
@@ -149,6 +157,12 @@ pub struct AssetOutputs {
     pub asset_id: AssetId,
 }
 
+#[derive(Serialize, Deserialize, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct AssetStringOutputs {

```

Comment:
Since this is the only resource so far with a GUID, let's just keep the name specific for now. We can generalize later if it makes sense to. Can we rename this to `NotificationOutputs`?